src/compatTest/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -180,24 +180,8 @@ sourceSets {
         return GradleRunner.create()
                 .withProjectDir(testProjectDir.getRoot())
                 .withArguments(arguments)
-                .withGradleVersion(getGradleVersion())
+                .withGradleVersion(System.getProperty("compat.gradle.version"))
```

Comment:
should we have a default version here in case the system property is not set? (e.g. when running from Intellij)

Reply:
I think there is a way in Intellij to select the version you want to use to run the tests. Last night I clicked on the run button at the left side of the class signature and I could see a dropdown to select the version. I tried a rerun and the dropdown didn't show up anymore...but I'm pretty sure there's a way to configure it :D

Reply:
Oh the other hand, it is stutter who will provide the property value per each compatTest task, if that property is not provided there is definitely something wrong in the project configuration and it would be something we need to look into. I rather see it fail than having it working silently using a default gradle version when we thing the tests are being run using different versions.